docker/Dockerfile.server

```diff
@@ -6,6 +6,11 @@
 # for why --platform
 FROM --platform=linux/amd64 python:3.9-bullseye
 
+RUN apt-get -y update
+RUN apt-get -y upgrade
+RUN apt-get install -y sqlite3
+COPY install_sqlite3.sh .
+RUN bash install_sqlite3.sh
```

Comment:
What would be the difference between `RUN apt-get install -y sqlite3` and `RUN bash install_sqlite3.sh` ? 

Can we choose just the latter?